### ISSUE TITLE
False positive PT019 with monkey patching and _ in test signature

### ISSUE THREAD

**Issue by jamesbraza** [m-burst/flake8-pytest-style#213]
# Bug report

## What's wrong

```python
from unittest.mock import patch


@patch("time.sleep")
def test_foo(_) -> None:
    _ = 0
```

Running `flake8-pytest-style==1.6.0` on this, it reports:

```
a.py:5:1: PT019 fixture _ without value is injected as parameter, use @pytest.mark.usefixtures instead
```

## How it should work

I think this should not be getting `PT019`, as I:
- Am required to have something in the signature for the patched `MagicMock`
- But I don't plan on using it, so I named it `_`
- Don't have a fixture `_` declared in the module

## System information

* Operating system: macOS Monterey version 12.6
* Python version: 3.10.9
* flake8 version: 6.0.0
* flake8-pytest-style version: 1.6.0

**Comment by jamesbraza** [comment]
Actually, I believe this is a dup of https://github.com/m-burst/flake8-pytest-style/issues/202, so if you want to close this as a duplicate, feel free.

**Comment by m-burst** [comment]
Yes, this is a dup of #202
